File: matiasmasso/Mat.NET/DT2/DTODeliveryHeader.vb
Public Class DTODeliveryHeader
    Property Guid As Guid
    Property Id As Integer
    Property Fch As Date
    Property Customer As DTOCustomer
    Property ImportAdicional As DTOAmt
    Property Import As DTOAmt
    Property Transmisio As DTOTransmisio
    Property Invoice As DTOInvoice
    Property Cod As DTOPurchaseOrder.Codis
    Property CashCod As DTO.DTOCustomer.CashCodes
    Property PortsCod As DTO.DTOCustomer.PortsCodes
    Property Facturable As Boolean
    Property Transportista As DTOTransportista
    Property Tracking As String
    Property EtiquetesTransport As DTODocFile
    Property UsrLog As DTOUsrLog2


    Public Class DTOCustomer
        Property Guid As Guid
        Property FullNom As String
    End Class

    Public Class DTOAmt
        Property eur As Decimal
    End Class

    Public Class DTOTransmisio
        Property Guid As Guid
        Property id As Integer
    End Class

    Public Class DTOInvoice
        Property Guid As Guid
        Property num As Integer

    End Class

    Public Class DTOUsrLog
        Property usrCreated As DTOUser

    End Class
    Public Class DTOUser
        Property Guid As Guid
        Property emailAddress As String
        Property nickname As String
    End Class

    Public Class DTOTransportista
        Property Guid As Guid
        Property abr As String
    End Class

    Public Class DTODocFile
        Property hash As String
    End Class
End Class

File: matiasmasso/Mat.NET/DTO/DTODeliveryHeader.cs
using System;

namespace DTO
{
    public class DTODeliveryHeader
    {
        public Guid Guid { get; set; }
        public int Id { get; set; }
        public DateTime Fch { get; set; }
        public DTOCustomer Customer { get; set; }
        public DTOAmt ImportAdicional { get; set; }
        public DTOAmt Import { get; set; }
        public DTOTransmisio Transmisio { get; set; }
        public DTOInvoice Invoice { get; set; }
        public DTOPurchaseOrder.Codis Cod { get; set; }
        public DTO.DTOCustomer.CashCodes CashCod { get; set; }
        public DTO.DTOCustomer.PortsCodes PortsCod { get; set; }
        public bool Facturable { get; set; }
        public DTOTransportista Transportista { get; set; }
        public string Tracking { get; set; }
        public DTODocFile EtiquetesTransport { get; set; }
        public DTOUsrLog2 UsrLog { get; set; }


        public class DTOCustomer
        {
            public Guid Guid { get; set; }
            public string FullNom { get; set; }
        }

        public class DTOAmt
        {
            public decimal eur { get; set; }
        }

        public class DTOTransmisio
        {
            public Guid Guid { get; set; }
            public int id { get; set; }
        }

        public class DTOInvoice
        {
            public Guid Guid { get; set; }
            public int num { get; set; }
        }

        public class DTOUsrLog
        {
            public DTOUser usrCreated { get; set; }
        }
        public class DTOUser
        {
            public Guid Guid { get; set; }
            public string emailAddress { get; set; }
            public string nickname { get; set; }
        }

        public class DTOTransportista
        {
            public Guid Guid { get; set; }
            public string abr { get; set; }
        }

        public class DTODocFile
        {
            public string hash { get; set; }
        }
    }
}
